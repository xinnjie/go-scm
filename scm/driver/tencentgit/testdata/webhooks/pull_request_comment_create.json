{
  "object_kind": "note",
  "user": {
    "name": "git_user2",
    "username": "git_user2",
    "avatar_url": "https://blog.bobo.com.cn/s/blog_6e572cd60101qls0.html"
  },
  "project_id": 229,
  "repository": {
    "name": "rename",
    "url": "https://tencent.com/z-413/rename.git",
    "description": "xx",
    "homepage": "https://tencent.com/z-413/rename"
  },
  "object_attributes": {
    "id": 294,
    "note": "back",
    "noteable_type": "MergeRequest",
    "author_id": 29,
    "created_at": "2015-03-13T09:52:25+0000",
    "updated_at": "2015-03-13T09:52:25+0000",
    "project_id": 14,
    "attachment": null,
    "line_code": null,
    "commit_id": "",
    "noteable_id": 33,
    "system": false,
    "st_diff": null,
    "url": "https://tencent.com/z-413/rename/merge_requests/6##note_33"
  },
  "merge_request": {
    "id": 2,
    "target_branch": "fenzhi01",
    "source_branch": "master",
    "source_project_id":14,
    "author_id": 11322,
    "assignee_id": 33,
    "title": "xx",
    "created_at": "2015-03-13T09:52:25+0000",
    "updated_at": "2015-03-13T09:52:25+0000",
    "milestone_id": 23,
    "state": "opened",
    "merge_status": "go",
    "target_project_id": 14,
    "iid": 3,
    "description": "Revision, not careful.",
    "position": 0,
    "locked_at": null,
    "source": {
      "name": "rename",
      "ssh_url": "git@tencent.com:z-413/rename.git",
      "http_url": "http://tencent.com:z-413/rename.git",
      "web_url": "https://tencent.com:z-413/rename",
      "namespace": "z-413",
      "visibility_level": 10
    },
    "target": {
      "name": "rename",
      "ssh_url": "git@tencent.com:z-413/rename.git",
      "http_url": "http://tencent.com:z-413/rename.git",
      "web_url": "https://tencent.com:z-413/rename",
      "namespace": "z-413",
      "visibility_level": 10
    },
    "last_commit": {
      "id": "1480a4610ca01dd10cb5bc30d8a1b1eab98c09a1",
      "message": "xxx\n",
      "timestamp": "2015-03-15T09:51:06+0000",
      "url": "https://tencent.com:z-413/rename/commit/1480a4610ca01dd10cb5bc30d8a1b1eab98c09a1",
      "author": {
        "name": "git_user2",
        "email": "git_user2@tencent.com"
      }
    }
  }
}