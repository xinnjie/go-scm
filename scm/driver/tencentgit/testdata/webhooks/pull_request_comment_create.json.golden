{
  "Action": "created",
  "Repo": {
    "ID": "4861503",
    "Namespace": "gitlab-org",
    "Name": "hello-world",
    "FullName": "gitlab-org/hello-world",
    "Perm": null,
    "Branch": "master",
    "Private": false,
    "Clone": "https://gitlab.com/gitlab-org/hello-world.git",
    "CloneSSH": "git@gitlab.com:gitlab-org/hello-world.git",
    "Link": "https://gitlab.com/gitlab-org/hello-world",
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
  },
  "PullRequest": {
    "Number": 1,
    "Title": "Update readme",
    "Body": "adding build instructions to readme",
    "State": "open",
    "Sha": "c4c79227ed610f1151f05bbc5be33b4f340d39c8",
    "Ref": "refs/merge-requests/1/head",
    "Source": "feature",
    "Target": "master",
    "Base": {
      "Ref": "master",
      "Repo": {
        "ID": "4861503",
        "Namespace": "gitlab-org",
        "Name": "hello-world",
        "FullName": "gitlab-org/hello-world",
        "Branch": "master",
        "Private": false,
        "Clone": "https://gitlab.com/gitlab-org/hello-world.git",
        "CloneSSH": "git@gitlab.com:gitlab-org/hello-world.git",
        "Link": "https://gitlab.com/gitlab-org/hello-world"
      }
    },
    "Head": {
      "Ref": "feature",
      "Sha": "c4c79227ed610f1151f05bbc5be33b4f340d39c8",
      "Repo": {
        "ID": "4861503",
        "Namespace": "gitlab-org",
        "Name": "hello-world",
        "FullName": "gitlab-org/hello-world",
        "Branch": "master",
        "Private": false,
        "Clone": "https://gitlab.com/gitlab-org/hello-world.git",
        "CloneSSH": "git@gitlab.com:gitlab-org/hello-world.git",
        "Link": "https://gitlab.com/gitlab-org/hello-world"
      }
    },
    "Fork": "gitlab-org/hello-world",
    "Link": "https://gitlab.com/gitlab-org/hello-world/merge_requests/1",
    "Closed": false,
    "Merged": false,
    "Author": {
      "ID": 51764,
      "Login": "sytses",
      "Name": "Sid Sijbrandij",
      "Email": "",
      "Avatar": "https://secure.gravatar.com/avatar/8c58a0be77ee441bb8f8595b7f1b4e87?s=80&d=identicon"
    },
    "Created": "2017-12-10T17:01:11Z",
    "Updated": "2017-12-10T17:05:14Z"
  },
  "Comment": {
    "ID": 50772616,
    "Author": {
      "ID": 51764,
      "Login": "sytses",
      "Name": "Sid Sijbrandij",
      "Email": "",
      "Avatar": "https://secure.gravatar.com/avatar/8c58a0be77ee441bb8f8595b7f1b4e87?s=80&d=identicon"
    },
    "Body": "lgtm",
    "Created": "2017-12-10T17:05:14Z",
    "Updated": "2017-12-10T17:05:14Z"
  },
  "Sender": {
    "ID": 51764,
    "Login": "sytses",
    "Name": "Sid Sijbrandij",
    "Email": "",
    "Avatar": "https://secure.gravatar.com/avatar/8c58a0be77ee441bb8f8595b7f1b4e87?s=80&d=identicon"
  }
}
